src/main/groovy/io/seqera/wave/service/k8s/K8sServiceImpl.groovy

```diff
@@ -585,7 +585,7 @@ class K8sServiceImpl implements K8sService {
      *      The {@link V1Job} description the submitted job
      */
     @Override
-    V1Job transferJob(String name, String containerImage, List<String> args, BlobCacheConfig blobConfig) {
+    V1Job launchJob(String name, String containerImage, List<String> args, BlobCacheConfig blobConfig) {
```

Comment:
maybe in the separate PR, but this method needs to be refactored to be generic for all the jobs, once scan and build are implemented, we can have separate  spec methods and can be invoked by check the type, but this method needs to be generic

Reply:
Agree. Let's keep into next PR